TheFourFifths/consus
# Delete item functional tests

### Summary

Functional test for deleting an item using the server api

### Checklist


### How to Review

1. Pull
2. npm install if you need to
3. npm test
4. Check that delete item functional tests ran and passed.
5. Verify tests are testing.

### Links

- [TFF-140](https://msoese.atlassian.net/browse/TFF-140)
